templates/example.html

```diff
@@ -13,9 +13,12 @@
   <body>
     <div class="body">
       {{> header.html}}
-      {{#sections}}
+      {{#data.sections}}
       <div class="box">
         <div class="column doc {{^inBody}}hide-on-mobile{{/inBody}}">
+          {{#data.experimental}}
+            {{> experiment.html}}
```

Comment:
@sebastianbenz here, it where I inject experiment.html and it does not render {{component}}, etc.